app/kubernetes/views/applications/create/createApplicationController.js

```diff
@@ -399,37 +400,37 @@ class KubernetesCreateApplicationController {
   }
 
   restorePersistedFolder(index) {
-    this.formValues.PersistedFolders[index].NeedsDeletion = false;
+    this.formValues.PersistedFolders[index].needsDeletion = false;
     this.validatePersistedFolders();
   }
 
   resetPersistedFolders() {
     this.formValues.PersistedFolders = _.forEach(this.formValues.PersistedFolders, (persistedFolder) => {
-      persistedFolder.ExistingVolume = null;
-      persistedFolder.UseNewVolume = true;
+      persistedFolder.existingVolume = null;
+      persistedFolder.useNewVolume = true;
     });
     this.validatePersistedFolders();
   }
 
   removePersistedFolder(index) {
-    if (this.state.isEdit && this.formValues.PersistedFolders[index].PersistentVolumeClaimName) {
-      this.formValues.PersistedFolders[index].NeedsDeletion = true;
+    if (this.state.isEdit && this.formValues.PersistedFolders[index].persistentVolumeClaimName) {
+      this.formValues.PersistedFolders[index].needsDeletion = true;
     } else {
       this.formValues.PersistedFolders.splice(index, 1);
     }
     this.validatePersistedFolders();
   }
 
   useNewVolume(index) {
-    this.formValues.PersistedFolders[index].UseNewVolume = true;
-    this.formValues.PersistedFolders[index].ExistingVolume = null;
-    this.state.persistedFoldersUseExistingVolumes = !_.reduce(this.formValues.PersistedFolders, (acc, pf) => acc && pf.UseNewVolume, true);
+    this.formValues.PersistedFolders[index].useNewVolume = true;
+    this.formValues.PersistedFolders[index].existingVolume = null;
+    this.state.persistedFoldersUseExistingVolumes = _.every(this.formValues.PersistedFolders, { useNewVolume: false });
     this.validatePersistedFolders();
   }
 
   useExistingVolume(index) {
-    this.formValues.PersistedFolders[index].UseNewVolume = false;
-    this.state.persistedFoldersUseExistingVolumes = _.find(this.formValues.PersistedFolders, { UseNewVolume: false }) ? true : false;
+    this.formValues.PersistedFolders[index].useNewVolume = false;
+    this.state.persistedFoldersUseExistingVolumes = _.every(this.formValues.PersistedFolders, { useNewVolume: false });
```

Comment:
looked at `every` definition and feels like it is incorrect here.

Reply:
nice find, I'll use _.some